src/animation.js

```diff
@@ -15,6 +15,7 @@ class AnimationHandle {
 		this.speed = this._speed = 1.0;
 		this._lastChange = null;
 		this._lastChangeX = null;
+		this._animationNaturalSpeed = animation.naturalSpeed;
```

Comment:
Should  `naturalSpeed` have a default value here, such as `1.0`?

Reply:
Yes it should, I just forgot to add that.